diff --git a/.SRCINFO b/.SRCINFO
index ceb858a..45ddcce 100644
--- a/.SRCINFO
+++ b/.SRCINFO
@@ -27,12 +27,16 @@ pkgbase = sogo
 	backup = etc/sogo/sogo.conf
 	backup = etc/httpd/conf/extra/SOGo.conf
 	backup = etc/conf.d/sogo
 	source = http://www.sogo.nu/files/downloads/SOGo/Sources/SOGo-4.3.2.tar.gz
+	source = patch-gnustep-base-1.27.patch::https://sogo.nu/bugs/file_download.php?file_id=2190&type=bug
+	source = gcc10-fix.patch::https://sogo.nu/bugs/file_download.php?file_id=2189&type=bug
 	source = sogo.service
 	source = sogo.confd
 	source = sogo_configure.patch
 	sha256sums = ecf22e7763a3113ac86a891abbd0e50f7eac19275428798c34cf742cc83a446b
+	sha256sums = 663a9887ce7b31d6d6f4a54e7fe2d84dc2a569a5844a3194f8b9d7ee479f7c67
+	sha256sums = 3ed561519ad2a635869dd1d961329b557e1fa8fff0b0c4bc7e0b40926a35b13a
 	sha256sums = 0720b9ad35a05d86d794c7adbf18277ecde57ed147e96f6105acca93f19d3b8c
 	sha256sums = 8ee0d1ad77e998ea801053fce175d8c4a1c55dcc5ee1ff78f0a8e3797187a6a7
 	sha256sums = e64ea4aa0ddf29785de8d786ab7ab09f940bfe316b6f1deeb8d04d9d16d35db1
 
diff --git a/PKGBUILD b/PKGBUILD
index 1828037..8876087 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -12,9 +12,9 @@ license=('GPL')
 options=('!strip')
 replaces=('sogo2')
 install=sogo.install
 makedepends=('gcc-objc'
-             'gnustep-make<=1.26.0')
+             'gnustep-make')
 depends=("sope>=${pkgver}" 
          'gnustep-base'
          'libmemcached'
          'memcached'
@@ -31,19 +31,25 @@ optdepends=('postgresql: run database server for sogo locally'
 backup=('etc/sogo/sogo.conf'
         'etc/httpd/conf/extra/SOGo.conf'
         'etc/conf.d/sogo')
 source=("http://www.sogo.nu/files/downloads/SOGo/Sources/SOGo-${pkgver}.tar.gz"
+        "patch-gnustep-base-1.27.patch::https://sogo.nu/bugs/file_download.php?file_id=2190&type=bug"
+        "gcc10-fix.patch::https://sogo.nu/bugs/file_download.php?file_id=2189&type=bug"
         "sogo.service"
         "sogo.confd"
         "sogo_configure.patch")
 sha256sums=('ecf22e7763a3113ac86a891abbd0e50f7eac19275428798c34cf742cc83a446b'
+            '663a9887ce7b31d6d6f4a54e7fe2d84dc2a569a5844a3194f8b9d7ee479f7c67'
+            '3ed561519ad2a635869dd1d961329b557e1fa8fff0b0c4bc7e0b40926a35b13a'
             '0720b9ad35a05d86d794c7adbf18277ecde57ed147e96f6105acca93f19d3b8c'
             '8ee0d1ad77e998ea801053fce175d8c4a1c55dcc5ee1ff78f0a8e3797187a6a7'
             'e64ea4aa0ddf29785de8d786ab7ab09f940bfe316b6f1deeb8d04d9d16d35db1')
 
 prepare() {
   cd "$srcdir/SOGo-${pkgver}"
   patch configure ../sogo_configure.patch
+  patch -p1 <"$srcdir/gcc10-fix.patch"
+  patch -p1 <"$srcdir/patch-gnustep-base-1.27.patch"
 }
 
 build() {
   cd "$srcdir/SOGo-${pkgver}"
